src/stacks-hierarchy/Root_ParkingViolationsReporting/Root_ParkingViolationsQr.tsx

```diff
@@ -138,6 +141,7 @@ export const Root_ParkingViolationsQr = ({
     <ScreenContainer
       title={t(ParkingViolationTexts.qr.title)}
       secondaryText={t(ParkingViolationTexts.qr.instructions)}
+      leftHeaderButton={isLoading ? {type: 'none'} : {type: 'back'}}
```

Comment:
Why not just `isLoading ? undefined : {type: 'back'}`. I think it is slightly preferable to have a single method for specifying that it should be no header button, and not both `undefined` and `{type: none}`.

Reply:
Because of the default in `ScreenContainer`. The alternative is to remove that default and add it explicitly to the other screens using `ScreenContainer`.

Reply:
I agree and decided to remove the default and rather add it explicitly to the other screens.